README.md

```diff
@@ -85,6 +84,49 @@ $ helm install mypega pega/pega --namespace mypega --values pega.yaml
 ```bash
 $ helm delete mypega
 ```
+# Downloading Docker images for your deployment
+
+Clients with appropriate licenses can request access to several required images from the Pega-managed Docker image repository. With your access key, you can log in to the image repository and download docker images that are required to install the Pega Platform onto your database. After you pull the images to your local system you must push them into your private Docker registry.
+
+For details, see the examples listed in the runbooks:
+
+* [Preparing your local Linux system](docs/prepping-local-system-runbook-linux.md)
+* [Preparing your local Windows 10 system](docs/prepping-local-system-runbook-windows.md)
+
+Pegasystems uses a standard naming practice of hostname/product/image:tag. All Pega images are available from the pega-docker.downloads.pega.com host. The :tag represents the version if Pega being deployed, for example :8.3.1 to download Pega 8.3.1. Pega maintains three types of required Docker images for Client-managed Cloud deployments of Pega Platform:
+
+ Name        | Description                                           |
+-------------|-------------------------------------------------------|
+platform/pega  | *Download required. Deploys the Pega Platform with its customized version of the Tomcat application server |
```

Comment:
Is the * at the start of this line intentional or a formatting problem?

Reply:
it's intentional...I'm fine with removing